README.md

```diff
@@ -7,16 +7,41 @@ For a quick start follow the instructions below. For more in depth information o
 ## Concept
 We build a service request by using a `Bar` class as the point where you fire your `Call` and get a `Result`.
 
-## Get Served
-
-Take a look at the `BarSpec`, in short:
+## Perform a Call
 
+Take a look at the `ServiceSpec`, in short:
+*To get model object*
 ```swift
-
+        let service = Service(configuration: Configuration(baseURL: "http://jsonplaceholder.typicode.com")
+        let call = Call(path: "posts")
+
+        service.perform(call, toModelResult: { (result: Result<Posts>) in
```

Comment:
❓Hebben we het raw JSON result eigenlijk echt nodig? Mss moeten we maar forceren om een mapping model te maken dan?